### PR TITLE
FixforIssue#49 - Adding litmus demo video to the README.md

### PR DESCRIPTION
**Pull request by yudaykiran**
What this PR does?
- Adds a demo video to the litmus readme file.
- Created new folder "images" containing animated asciinema images.

Signed-off-by: Uday Kiran Y <uday.kiran@openebs.io>

### CODE REVIEW

**Comment by ksatchit** [comment]
Nice! Can we also have the "build:pass" badge shifted to top of the readme

**Comment by AmitKumarDas** [comment]
thanks @yudaykiran for the videos.

I felt the videos are not in sync.
Perhaps a single video is better at making contributors to understand the principles of litmus.
The pace of video can also be increased a bit.

**Comment by yudaykiran** [comment]
@AmitKumarDas the initial idea was to group them together as a single video, but then we decided on having separate videos for the status. We will have to group them together if we want to see some sync between the different terminals. I have got an idea where we might be able to increase the pace of the video, will try it out.

**Comment by yudaykiran** [comment]
@kmova @AmitKumarDas review comments addressed in the new commit.

File: README.md [hunk 1]
```diff
@@ -79,6 +81,10 @@ Feature: MySQL services are not affected due to node failures.
     And verify MySQL-client can still access data.
 ```
 
+<p align="center">
```
Comment (kmova) [inline README.md]: This video looks good!

Can you move this to a separate section -- named "Demo : <title...>". Place this section before .. the section for "Running a specific Test"

**Comment by yudaykiran** [comment]
@kmova resolved the review comments.

**Pull request merged**